diff --git a/dwt.c b/dwt.c
index f73d7d7..1b1151b 100644
--- a/dwt.c
+++ b/dwt.c
@@ -244,8 +244,7 @@ term_char_size_changed (VteTerminal *vtterm,
                                    GTK_WIDGET (vtterm),
                                    &geometry,
                                    GDK_HINT_MIN_SIZE |
-                                   GDK_HINT_BASE_SIZE |
-                                   GDK_HINT_RESIZE_INC);
+                                   GDK_HINT_BASE_SIZE);
     gtk_widget_queue_resize (GTK_WIDGET (vtterm));
 }
 
